.github/workflows/test-toolkits.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Toolkits
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      tool_matrix: ${{ steps.dataStep.outputs.tools }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get Toolkits
+        id: dataStep
+        run: |
+          TARGETS=$(./.github/scripts/get_toolkits.sh)
+          echo "tools=$(jq -cn --argjson environments "$TARGETS" '{target: $environments}')" >> $GITHUB_OUTPUT
+
+  test-toolkits:
+    needs: setup
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJson(needs.setup.outputs.tool_matrix) }}
+    steps:
+    - run: echo ${{ matrix.target }}
+
+    - name: Checkout code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+        cache: 'pip'
+
+    - name: Install Python Dependencies
```

Comment:
Since arcade-ai and toolkits will no longer be released at the same time, I think we should install the latest release of arcade-ai.

Consider the case where I create a new toolkit that uses a new function in the SDK that hasn't been released yet. The toolkit would pass the tests, but then once released, it would fail whenever someone tries to use the tools.

Reply:
true, you wanna change that?